scripts/main.py

```diff
@@ -0,0 +1,232 @@
+import os
+import time 
+import pickle
+import random
+import numpy as np
+import matplotlib.pyplot as plt
+import torch
+from multiprocessing import Pool
+import vboc.model as models
+from vboc.parser import Parameters, parse_args
+from vboc.abstract import SimDynamics
+from vboc.controller import ViabilityController
+from vboc.learning import NeuralNetwork, RegressionNN, plot_viability_kernel
+
+
+def computeDataOnBorder(N_guess, test_flag=0):
+    valid_data = np.empty((0, model.nx))
+    controller.resetHorizon(N_guess)
+
+    # Randomize the initial state
+    d = np.array([random.uniform(-1, 1) for _ in range(model.nv)])
+    q = np.array([
+        random.uniform(params.q_min + model.eps, params.q_max - model.eps)
+        for _ in range(model.nq)]
+    )
+
+    # Set the initial guess
+    x_guess = np.zeros((N_guess, model.nx))
+    u_guess = np.zeros((N_guess, model.nu))
+    x_guess[:, :model.nq] = np.full((N_guess, model.nq), q)
+
+    if not test_flag:
```

Comment:
`test_flag` is not a good name for this variable.

Reply:
I change it into `uniform`